hvac/api/auth_methods/userpass.py

```diff
@@ -22,10 +23,13 @@ def create_or_update_user(self, username, password, mount_point=DEFAULT_MOUNT_PO
         :type username: str | unicode
         :param password: The password for the user. Only required when creating the user.
         :type password: str | unicode
+        :param policies: The list of policies to be set on username created.
+        :type policies: str | unicode
         :param mount_point: The "path" the method/backend was mounted on.
         :type mount_point: str | unicode
         """
         params = {
+            'policies': policies,
```

Comment:
According to the API docs this should instead be `token_policies`, no?: https://www.vaultproject.io/api/auth/userpass/index.html#inlinecode-token_policies-7

Reply:
But in https://www.vaultproject.io/api/auth/userpass/index.html#sample-payload it shows "policies"

Reply:
It's explained here https://github.com/hashicorp/vault/blob/master/CHANGELOG.md#120-july-30th-2019 this PR tries to adapt hvac to this change : https://github.com/hvac/hvac/pull/563